frigate/api/defs/regenerate_query_parameters.py

```diff
@@ -0,0 +1,7 @@
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class RegenerateQueryParameters(BaseModel):
+    source: Optional[str] = "thumbnails"
```

Comment:
this type here should probably use the enum type

Reply:
oh for sure

Reply:
I mean the type as well, so the validation is applied